main/src/ca/uwaterloo/flix/language/errors/WeederError.scala

```diff
@@ -679,6 +679,30 @@ object WeederError {
 
   }
 
+  /**
+    * An error raised to indicate that the case of an alias does not match the case of the original value.
+    *
+    * @param patt     the invalid regular expression
+    * @param loc      the location where the error occurred
+    */
+  case class InvalidRegularExpression(patt: String, loc: SourceLocation) extends IllegalLiteral {
```

Comment:
We should add handful of tests to `TestWeeder`.

Reply:
I've added four tests that rightfully fail due to bad literals.